tests/test_e2e_miniscript.py

```diff
@@ -83,6 +83,9 @@ def run_test_e2e(wallet_policy: WalletPolicy, core_wallet_names: List[str], rpc:
     result = multisig_rpc.walletcreatefundedpsbt(
         outputs={
             out_address: Decimal("0.01")
+        },
+        options={
+            "changePosition": 1
```

Comment:
Still not a big fan of this; it would be nice to have the change address in a random position.

At least, it would be nice to add a comment explaining why this is here (probably copying the comment in `tests/test_e2e_multisig.py` and `tests/test_e2e_tapscripts.py`; refactoring the common code could be a future PR).

Reply:
I can duplicate the tests to test all the possible positions if you prefer :shrug:

Reply:
I think just a one-line comment explaining why it's there (e.g. what breaks without, and why if you know it) would do. Can improve it in a separate future PR.